custom_components/proxmoxve/sensor.py

```diff
@@ -90,6 +90,39 @@ class ProxmoxSensorEntityDescription(ProxmoxEntityDescription, SensorEntityDescr
         suggested_unit_of_measurement=UnitOfInformation.GIGABYTES,
         entity_registry_enabled_default=False,
     ),
+    ProxmoxSensorEntityDescription(
+        key=ProxmoxKeyAPIParse.DISK_ZFS_TOTAL,
+        name="Disk ZFS total",
+        icon="mdi:harddisk",
+        native_unit_of_measurement=UnitOfInformation.BYTES,
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+        suggested_unit_of_measurement=UnitOfInformation.GIGABYTES,
+        entity_registry_enabled_default=False,
+    ),
+    ProxmoxSensorEntityDescription(
+        key=ProxmoxKeyAPIParse.DISK_ZFS_USED,
+        name="Disk ZFS used",
+        icon="mdi:harddisk",
+        native_unit_of_measurement=UnitOfInformation.BYTES,
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+        suggested_unit_of_measurement=UnitOfInformation.GIGABYTES,
+        entity_registry_enabled_default=False,
+    ),
+    ProxmoxSensorEntityDescription(
+        key=ProxmoxKeyAPIParse.DISK_ZFS_AVAIL,
+        name="Disk ZFS available",
+        icon="mdi:harddisk",
+        native_unit_of_measurement=UnitOfInformation.BYTES,
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+        suggested_unit_of_measurement=UnitOfInformation.GIGABYTES,
+        entity_registry_enabled_default=False,
+    ),    
```

Comment:
As this should be a new category for the configuration entry, this should be added to `PROXMOX_SENSOR_STORAGE` and use a proper coordinator.